libs/s3/s3_multipart_test_lib.py

```diff
@@ -132,7 +132,7 @@ def upload_parts(self, mpu_id: int = None, bucket_name: str = None, object_name:
         :return: (Boolean, List of uploaded parts).
         """
         try:
-            b_size = kwargs.get("block_size", "1M")
+            b_size = int(kwargs.get("block_size", "1048576"))
```

Comment:
This change will impact all multipart tests as this variable is introduced recently by @aditipande96 and her changes looks good to me as those are not impacting s3 tests. We can go with ur changes as well but then you need to do RVF for all(~30+) tests getting impacted because of this change.

Reply:
Reverted changes.